use-case-guides/workday-upcoming-birthdays-using-reporting/README.md

```diff
@@ -0,0 +1,162 @@
+time_in_minutes: 10
+difficulty_level: Beginner
+---
+
+# Use Case Guide: Workday: Using reporting to query for upcoming birthdays
+
+## **Introduction**
+
+Workday's Reports-as-a-Service (RaaS) is a powerful feature that revolutionizes data access and integration in the Workday ecosystem. By allowing custom reports to be exposed as web services, RaaS facilitates seamless data interactions with external systems. 
+
+By bringing these reports into Creator Studio, we can add a conversational interface to them, making it easier than ever for your users to get access to the data that lies within Workday.
+
+In this tutorial, we will explore how to call RaaS from Creator Studio. To guide us, we will be using a pre-built report that gets the upcoming birthdays.
+
+Let's get started!
+
+# What are we building?
+
+## Conversation Design
+
+[This purple chat]() shows the experience we are going to build.
+
+**This use case is made up of:**
+
+✨ **Triggers:** Natural Language
+
+🤲 **Slots**: None
+
+🏃‍♂️ **Actions:** Retrieve data from workday report
+
+📜 **Response Data:** The data from a RaaS report
+
+📚 **Guidelines:** Existing report with no lookup fields
```

Comment:
There's just ONE guideline -- "select fields to render from RaaS report" 
You can review all guidelines here: https://developer.moveworks.com/creator-studio/conversation-design/overview#guidelines